clustering/clustering_example.Rmd

```diff
@@ -52,10 +52,14 @@ add them as data.frames to your environment.
 
 ```{r Import data from .tsv files}
 # Read in metadata tsv file
-metadata <- readr::read_tsv(file.path("data", "metadata_GSE12955.tsv"))
+metadata <- readr::read_tsv(file.path("data", # Replace with path to your metadata file
```

Comment:
Also debating about if we want to declare these file paths else where, but I am less certain about this one.

Reply:
This is the only comment I did not address in the last commit because it is not clear that declaring these paths else where will be beneficial or would make a difference here as it did with the file paths to the plots/results directories. Was this also your thinking?

Reply:
Also, is your suggestion here to declare the metadata and data file paths else where or just the path to the data folder? I believe the latter could be useful but that's also assuming that users would save their metadata and other data files in the same directory.

Reply:
I think either strategy is fine. But perhaps just declaring the folder and building the metadata and data paths from that? As long as we have a comment at each place the person would need to change it, I'm not dead set on either way.